Lib/idlelib/idle_test/test_configdialog.py

```diff
@@ -44,7 +40,7 @@ def setUpModule():
 
 def tearDownModule():
     global root, configure
-    idleConf.userCfg = testcfg
+    idleConf.userCfg = usercfg
```

Comment:
I though maybe I branched after fetching that.  Anyway, I fetched again, pushed master to origin, and 
ran <git merge origin/master>, from the devguide, which has worked before. and pushed.
... Above -+ is gone from new diff.